core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1031,20 +1030,46 @@ class HttpGoogleServicesDAO(
     }
   }
 
+  override def addGoogleProjectLabel(googleProject: GoogleProjectId, key: String, value: String): Future[Unit] = {
+    implicit val service: GoogleInstrumentedService.Value = GoogleInstrumentedService.CloudResourceManager
+
+    val cloudResourceManagerServiceAccountCredential = getCloudResourceManagerServiceAccountCredential
+    val cloudResourceManager = getCloudResourceManager(cloudResourceManagerServiceAccountCredential)
+    for {
+      project <- getGoogleProject(googleProject)
+      _ = project.getLabels.put(key, value)
```

Comment:
I know I can probably Google this myself too and perhaps I will, but why for updating the labels, do we need to first call `getGoogleProject`, update the labels, and then `update` the project.  Whereas in `updateGoogleProjectName`, we do not get the project first and instead just create a new `Project` and specify the field we want updated?

Reply:
RBS gives us projects with existing labels, so I think this is the right call so we don't replace any existing labels.  I've added a comment about this to clarify